src/libraries/Trading.sol

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+ */
+
+pragma solidity ^0.4.13;
+
+library Trading {
```

Comment:
Hmm, I wonder how this actually works... I see in other files you are importing it and then doing `Trading.Bid` and `Trading.Ask`, but I thought libraries in Solidity needed to be linked in via an external address at compile time?  Maybe that doesn't apply for nested types like Enums?

Reply:
Good question -- I'm actually not sure either.  Do you think this should be done in a different way?

Reply:
No, if this works I'm generally happy with it.  I'm just surprised it works.